charts/nri-metadata-injection/templates/NOTES.txt

```diff
@@ -22,18 +21,3 @@ Check if New Relic environment variables were injected:
     NEW_RELIC_METADATA_KUBERNETES_NAMESPACE_NAME=default
     NEW_RELIC_METADATA_KUBERNETES_POD_NAME=busybox0
     NEW_RELIC_METADATA_KUBERNETES_CONTAINER_NAME=busybox
-
-{{- else -}}
-###############################################################
-####       ERROR: You did not set a cluster name.          ####
-###############################################################
```

Comment:
Now, when `cluster` is not set, installation fails. Therefore, I think this warning is not needed anymore